### PR TITLE
Standardise Storage and Loading Order for Client Notes

### LINKED ISSUE #319: Bug: Ensure order of Client Notes is maintained at exit and start-up
before closing: 
image

after opening again: 
image

### PR DESCRIPTION
**Pull request by rtshkmr**
## Description

As per title.

Fixes #319 

## Testing

NIL, just works man: 

image
image

## Remarks

NIL

### CODE REVIEW

File: src/main/java/seedu/address/model/TbmManager.java [hunk 1]
```diff
@@ -57,26 +54,13 @@ public void setClients(List<Client> clients) {
...
 
     /**
-     * Replaces all notes in TbmManager with the given list of notes.
+     * Replaces all country notes in TbmManager with the given list of country notes.
      *
-     * @param notes The given list of notes.
+     * @param countryNotes The given list of country notes.
      */
-    public void setNotes(List<Note> notes) {
-        countryNotesManager.clear();
```
Comment (raysonkoh) [inline src/main/java/seedu/address/model/TbmManager.java:65]: This is important!! don't delete this

File: src/main/java/seedu/address/storage/JsonSerializableTbmManager.java [hunk 2]
```diff
@@ -61,11 +61,10 @@ public TbmManager toModelType() throws IllegalValueException {
             }
             tbmManager.addClient(client);
         }
-        for (JsonAdaptedNote note: notes) {
+        for (JsonAdaptedNote note: countryNotes) {
```
Comment (raysonkoh) [inline src/main/java/seedu/address/storage/JsonSerializableTbmManager.java]: Renaming `note` to `countryNote` might be clearer

File: src/test/data/JsonSerializableTbmManagerTest/typicalClientsTbmManager.json [hunk 3]
```diff
@@ -75,21 +75,21 @@
       "timezone": "GMT-4",
       "contractExpiryDate": "2-8-2021"
     } ],
-  "notes": [ {
-      "contents": "lives in a pineapple",
-      "countryCode": "NULL_CC",
-      "tags": ["godLike"]
+  "countryNotes": [ {
+      "contents": "too hot, hot damn",
```
Comment (raysonkoh) [inline src/test/data/JsonSerializableTbmManagerTest/typicalClientsTbmManager.json:79]: Made me chuckle

**Review by raysonkoh — CHANGES_REQUESTED** [review]
Please add in `countryNotesManager.clear()` in `TbmManager#setCountryNotes` method. This is quite important. Otherwise I'm ok with the changes.

File: src/main/java/seedu/address/storage/JsonSerializableTbmManager.java (hunk 2 above)
Comment (qwoprocks) [reply src/main/java/seedu/address/storage/JsonSerializableTbmManager.java]: renamed to `jsonAdaptedCountryNote`

File: src/main/java/seedu/address/model/TbmManager.java (hunk 1 above)
Comment (qwoprocks) [reply src/main/java/seedu/address/model/TbmManager.java:65]: my bad, added it back

**Review by raysonkoh — APPROVED** [review]
LGTM

**Pull request merged**